rpcs3/Emu/System.cpp

```diff
@@ -1700,7 +1700,11 @@ void Emulator::Run(bool start_playtime)
 	idm::check<named_thread<ppu_thread>>(ppu_thread::id_base, [](cpu_thread& cpu)
 	{
 		cpu.state -= cpu_flag::stop;
-		cpu.notify();
+
+		if (cpu.id_type() == 1)
+			thread_ctrl::raw_notify(static_cast<named_thread<ppu_thread>&>(cpu));
+		else if (cpu.id_type() == 2)
+			thread_ctrl::raw_notify(static_cast<named_thread<ppu_thread>&>(cpu));
```

Comment:
```suggestion
		else if (cpu.id_type() == 2)
			thread_ctrl::raw_notify(static_cast<named_thread<spu_thread>&>(cpu));
```

Reply:
wait actually its idm::check<named_thread<ppu_thread>> so its always PPU..

Reply:
if `cpu` type was named_thread<ppu_thread>& there wasno need for cast or checks in the first place.

Reply:
It' temporarily to test raw_notify, will remove.